app/views/tag/show.html.erb

```diff
@@ -55,71 +55,73 @@
+                            <span style="white-space: nowrap;">
+                                <span rel="tooltip" title="This 'tagname' forms part of the URL for this topic." class="badge badge-secondary" style="background: #ccc;color: white;">
+                                    <%= params[:id] %>
+                                </span>
+                                <a rel="tooltip" title="Show related topics" onclick="$('.related-tags').show();$(this).hide();"><i class="fa fa-plus-circle" style="vertical-align: middle;color:#ccc;"></i></a>
+                                <span class="related-tags" style="display:none;">
+                                    <% unless @related_tags.nil? %>
+                                        <% @related_tags.each do |tag| %>
+                                            <a href="/tag/<%= tag.name %>" class="badge badge-secondary" style="background: #aaa;color: white;">
+                                                <%= tag.name %>
+                                            </a>
+                                        <% end %>
+                                    <% end %>
+                                </span>
+                            </span>
+                        </div>
+                    </h1>
+
+                    <% if @wildcard %>
+
+                        <% if @node_type == "questions"%>
+                            <p><i>Showing <b>questions</b></i></p>
+                        <% elsif @node_type == "note" && params[:id].match("activity:") %>
+                            <p><i>Showing <b>activities</b></i></p>
+                        <% elsif @node_type == "note" && params[:id].match("replication:") %>
+                            <p><i>Showing <b>completed activities</b></i></p>
+                        <% end %>
+
+                        <p><i>This is a wildcard (*) tag search. Please enter a tag without wildcards to display lead wiki pages.</i></p>
+
+                    <% end %>
+
+                    <% if @wiki %>
+                        <p class = "wiki-sub wiki-summary" id="wiki-summary"><%= @wiki.latest.body.split('\n').first %></p>
+                    <% end %>
+
+                    <div class="tag-buttons">
+                        <div class="btn-group" role="group">
+                            <% if current_user && current_user.following(params[:id]) %>
+                                <a rel="tooltip" title="<%= translation('sidebar._post_button.share_your_work') %>" data-placement="bottom" href="/post?tags=<%= @title %>" class="btn btn-primary requireLogin">New post <i class="fa fa-plus fa-white"></i></a>
+                            <% else %>
+                                <a class="btn btn-primary requireLogin" target="_blank" href="/subscribe/tag/<%= params[:id] %>"> Follow</a>
+                                <% if @related_tags %>
```

Comment:
Oh no! This was my fault for sure. I accidentally committed this without final checks because I misunderstood the gitpod interface. I thought it was to push to a feature branch but it went to main! 😱

Thanks for catching and solving this!!!